org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  Bucket:
+    Type: String
+  Frequency:
+    Type: String
+    Default: "monthly"
+    AllowedValues:
+    - "monthly"
+    - "weekly"
+    - "hourly"
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub "${AWS::StackName}"
+      ReportVersioning: False
+      TimeUnit: !Ref Frequency
+      Compression: "parquet"
+      Format: "parquet"
+      S3Region: !Ref primaryRegion
```

Comment:
`primaryRegion` is not a CFN parameter, it's only available to orgf configs.  you can use the [CFN pseudo parameter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html) `!Ref AWS::Region`